components/Footer.js

```diff
@@ -9,7 +9,7 @@ import NavLink from './NavLink'
 export default function Footer (props) {
```

Comment:
Need to remove the "About" link too (see https://otp-admin-ui-git-fdot-tweaks-ibi-group.vercel.app/about)

Reply:
Fixed in c21b67c